### PR TITLE
Add developer info

### PR DESCRIPTION
**Pull request by yuhan6665**
- progress 75/327
- Add extra-info json. In the future, it can be used to record other things as well, like date of suspension, size of project
- Some raw data is put there, which helped finding the developer info

- Add code to auto generate developer section in readme
- ~~Currently only properties-tree contain developer info~~

### CODE REVIEW

**Comment by yuhan6665** [comment]
@MarkShawn2020

File: development/backend/nodejs/src/genMarkdown.ts [hunk 1]
```diff
@@ -80,7 +97,7 @@ export const genPropertiesMd = (style: MdProvinceStyle, sourcePath: string, targ
         let isStarted               = false
         let isEnded                 = false
         for (let line of linesRaw) {
-            if (isStarted && REG_END.test(line)) isEnded = true
+            if (isStarted && /^## 其他曝光/.test(line)) isEnded = true
```
Comment (MarkShawn2020) [inline development/backend/nodejs/src/genMarkdown.ts:100]: do not hardcode at here, you can replace the raw REG_END with your new tester

File: development/backend/nodejs/src/itree/ds.ts [hunk 2]
```diff
@@ -4,9 +4,12 @@
 export interface IItem {
     name: string
     uri: string
+    developer: string
 }
 
-export interface IFlatItem extends IItem {
+export interface IFlatItem {
```
Comment (MarkShawn2020) [inline development/backend/nodejs/src/itree/ds.ts:10]: why not put the `developer` field into `IFlatItem` at the same time

Comment (yuhan6665) [reply development/backend/nodejs/src/itree/ds.ts]: wasn't sure if we want to put extra info in there. I just update it. Thanks for your review!

**Comment by FeelBreeZe** [comment]
能帮我添加一下吗？
https://github.com/WeNeedHome/SummaryOfLoanSuspension/discussions/1134

**Comment by MarkShawn2020** [comment]
> 能帮我添加一下吗？ #1134

抱歉我精力有限，目前只负责维护本项目技术部分，您更新停贷项目相关的信息可以试试寻求一下其他维护者

**Comment by yuhan6665** [comment]
Thanks. I will fill more developer info. :)
BTW who can grant permission for this repo now? I can help merging others pr
cc @WeNeedHome

**Pull request merged**